docs/fixes/2026-06-15-name-template-ignore-missing-template-values.md

```
@@ -0,0 +1,119 @@
+templates:
+  settings:
+    ignore_missing_template_values: true
+```
+
+When `true`, Go template processing must **not** fail if a referenced variable is missing;
+the missing reference renders as `<no value>` instead of aborting with an error. This flag was
+added in PR #2158.
+
+Users who set this flag still hit hard errors like:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)